tests/dataimportcron_test.go

```diff
@@ -199,6 +199,33 @@ var _ = Describe("DataImportCron", func() {
 		table.Entry("[test_id:7414] Should successfully import PVC from registry URL on source digest update", scheduleOnceAYear, cdiv1.DataImportCronRetainAll, 2, false),
 		table.Entry("[test_id:7406] Should successfully garbage collect old PVCs when importing new ones", scheduleOnceAYear, cdiv1.DataImportCronRetainNone, 2, true),
 	)
+
+	It("[test_id:XXXX] should delete jobs on deletion", func() {
+		url := trustedRegistryURL()
+		noSuchCM := "nosuch"
+		cron = NewDataImportCron("cron-test", "5Gi", scheduleEveryMinute, dataSourceName, cdiv1.DataVolumeSourceRegistry{URL: &url, PullMethod: &registryPullNode, CertConfigMap: &noSuchCM}, cdiv1.DataImportCronRetainAll)
+		By(fmt.Sprintf("Create new DataImportCron %s", url))
+		cron, err = f.CdiClient.CdiV1beta1().DataImportCrons(f.Namespace.Name).Create(context.TODO(), cron, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify jobs were created")
```

Comment:
Should it also create pods associated with those jobs? AFAICT there are 3 levels of objects. Top level DataImportCrons, second level CronJob and Initial Job, and third level the actual pods created by those jobs. So I would expect the test to verify all 3 get created, and that all 3 get deleted when the top level is deleted.

Reply:
Added